admin-dev/themes/new-theme/js/pages/product/components/create-product-modal.ts

```diff
@@ -42,7 +42,7 @@ export default class CreateProductModal {
 
       const iframeModal = new FormIframeModal({
         id: 'modal-create-product',
-        formSelector: 'form[name="product"]',
+        formSelector: 'form[name="create_product"]',
```

Comment:
However I just tested the PR, it seems like the redirection after the submit doesn't work anymore I'm thinking maybe it's because of this modified selector The data returned by the `onFormLoaded` method are based on this form selector.

The problem here is that the name of the creation form and the name of the edition form are different so the selector can't work for both, so you're gonna need a selector that can work on both (not based on name). You could add a common `product-form` class on both and then use `form.product-form` as a common selector.